src/sentry/utils/samples.py

```diff
@@ -116,9 +116,6 @@ def create_sample_event(project, platform=None, default=None, raw=True,
     if not platform and not default:
         return
 
-    if platform:
```

Comment:
we didnt really need this code path anymore and it prevented react-native from working